spring-rabbit/src/test/resources/log4j2-test.xml

```diff
@@ -6,7 +6,7 @@
 		</Console>
 	</Appenders>
 	<Loggers>
-		<Logger name="org.springframework.amqp.rabbit" level="info"/>
+		<Logger name="org.springframework.amqp.rabbit" level="debug"/>
```

Comment:
Maybe we don't need this any more?
I mean you have checked logs locally - OK: no need to increase logging for `main`